frontend/templates/views/partials/roadshow-video.j2

```diff
@@ -0,0 +1,7 @@
+{% do doc.styles.addCssFile('css/components/molecules/roadshow-video.css') %}
```

Comment:
As this is possibly never used on any page except the roadshow page, either directly add it to the template or rename to a more generic partial.